src/main/java/com/icthh/xm/ms/scheduler/manager/SchedulingManager.java

```diff
@@ -267,6 +292,10 @@ private ScheduledFuture scheduleOneTimeTask(RunnableTask expirable, TaskDTO task
         } else {
             task.setEndDate(task.getStartDate());
         }
+        Task entity = taskRepository.findById(task.getId()).orElseThrow(() ->
+            new IllegalArgumentException("Entity is not found"));
```

Comment:
I propose to use BusinessException as produces http 400 error instead of 500 and provides message localization.